README.md

```diff
@@ -1,15 +1,29 @@
 # Metaverse Backpack Dressing Room Sample
+
 Example WebGL Demo of the Metaverse Backpack
 
-## Setup
+## Getting started
+
+### Setup
 
 `$ git clone github.com/metaverse-backpack/unity-dressing-room-sample`
 
 `$ cd unity-dressing-room-sample`
 
 `$ git submodule update --init --recursive`
 
-`$ git submodule update --init --recursive`
+### Build settings
+
+Open up the Unity project in the Unity Hub. If there is a popup asking if you want to open the project in safe mode, click on `Ignore`. This is because the project starts with a native build target causing dependencies not to be loaded. Change the Build target to WebGL by choosing `File -> Build Settings...`. Then select `WebGL` in the `Platform` selection panel. Ensure `Scene` is selected under `Scenes in Build`.
+
+### Build and run
+
+Select `File -> Build And Run` to start a unity development web server and run the project in a browser (`http://localhost:51326`).
```

Comment:
@tobias-petrasch Unity actually just selects an available port so this isn't consistent. Unfortunately, I don't think there's any way to explicitly set which port this runs on, so we might need to remove those redirect URI check restrictions in the backend temporarily if we want to support that.

Reply:
We could also use an external http server (e.g., npx http-server) with a fixed port. Instead of running build & run in Unity, we only run the build command. Don't think it makes sense to change the backend.